Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -61,6 +61,7 @@ installation of the ``kube-proxy`` add-on:
     .. code:: bash
 
       kubectl -n kube-system delete ds kube-proxy
+      kubectl -n kube-system delete cm kube-proxy 
```

Comment:
Could you add a comment saying that it works only on > k8s 1.19 (I assume you have tested it)?

Reply:
Done.
Yes tested it on 1.20.2 but was added in 1.19, from the changelog:
`Kubeadm: during 'upgrade apply', if the kube-proxy ConfigMap is missing, assume that kube-proxy should not be upgraded. Same applies to a missing kube-dns/coredns ConfigMap for the DNS server addon. Note that this is a temporary workaround until 'upgrade apply' supports phases. Once phases are supported the kube-proxy/dns upgrade should be skipped manually. (#89593, @neolit123) [SIG Cluster Lifecycle]`